unlock-app/src/components/interface/VerificationStatus.tsx

```diff
@@ -87,6 +87,11 @@ export const VerificationStatus = ({ data, sig }: Props) => {
     return <InvalidKey reason="This ticket has expired" />
   }
 
+  // The current owner does not match the signed owner
+  if (!unlockKey) {
+    return <InvalidKey reason="The current owner does not match the signature!" />
+  
```

Comment:
missing `}` here perhaps?

Reply:
Wow this should not be part of this PR. Removing it!